tools/update-readme.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview Script to update the README with sponsors details in all packages.
+ *
+ *   node tools/update-readme.js
+ *
+ * @author Milos Djermanovic
+ */
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+import { readFileSync, writeFileSync } from "node:fs";
+import got from "got";
+
+//-----------------------------------------------------------------------------
+// Data
+//-----------------------------------------------------------------------------
+
+const SPONSORS_URL =
+	"https://raw.githubusercontent.com/eslint/eslint.org/main/includes/sponsors.md";
+
+const README_FILE_PATH = "./README.md";
+
+//-----------------------------------------------------------------------------
+// Helpers
+//-----------------------------------------------------------------------------
+
+/**
+ * Fetches the latest sponsors from the website.
+ * @returns {Promise<string>}} Prerendered sponsors markdown.
+ */
+async function fetchSponsorsMarkdown() {
+	return got(SPONSORS_URL).text();
+}
+
+//-----------------------------------------------------------------------------
+// Main
+//-----------------------------------------------------------------------------
+
+(async () => {
```

Comment:
Is this IIFE needed?

Reply:
Not really, it just survived through multiple copy-pastes from the original code in eslint/eslint, which is in a cjs module so top-level await was not available. I removed it now.